src/components/Footer.jsx

```diff
@@ -7,19 +7,22 @@ import { isAuthenticated } from "../utils/isAuthenticated";
 const notLoggedButtons = () => {
   if (!isAuthenticated()) {
     return (
-      <div className="col-span1 sm:col-span-2 space-x-5 sm:space-x-10 2xl:flex-col items-center justify-center xl:justify-between xl:px-10 2xl:px-0 2xl:justify-center contents">
-        <Link
-          to="/login"
-          className="flex inline-flex items-end justify-center px-20 py-3.5 mb-5  md:inline-flex xl:inline-flex 2xl:mb-7  md:mb-7 text-xl font-semibold leading-6 text-white whitespace-nowrap bg-primary rounded-md shadow-button hover-scale tracking-add mt-5"
-        >
-          Iniciar Sesión
-        </Link>
-        <Link
-          to="/register"
-          className="flex inline-flex shadow-button items-end justify-center px-20 py-3.5 mb-5  text-xl font-semibold leading-6  xl:mb-8 md:mb-7 text-white whitespace-no-wrap bg-tertiary hover-scale rounded-md md:inline-flex xl:inline-flex mt-5 md:mt-4 xl:mt-4"
-        >
-          Regístrate
-        </Link>
+      <div className="sm:col-span-2 sm:space-x-8 2xl:flex-col items-center justify-center xl:justify-between xl:px-10 2xl:px-0 2xl:justify-center sm:contents xl:contents 2xl:contents">
+        <div className="flex flex-col sm:flex-inle sm:space-x-8 2xl:flex-col contents">
+          <Link
+            to="/login"
+            className="flex space-x-4  sm:inline-flex items-end justify-center px-20 py-3.5 mb-5  md:inline-flex xl:inline-flex 2xl:mb-7  md:mb-7  text-xl font-semibold leading-6 text-white whitespace-nowrap bg-primary rounded-md shadow-button hover-scale tracking-add mt-5 xl:mt-5"
+          >
+            Iniciar Sesión
+          </Link>
+          <Link
+            to="/register"
+            className="flex space-x-4  sm:inline-flex shadow-button items-end justify-center px-20 py-3.5 mb-5  text-xl font-semibold leading-6  xl:mb-8 md:mb-7 text-white whitespace-no-wrap bg-tertiary hover-scale rounded-md md:inline-flex xl:inline-flex mt-5 md:mt-4 mt-5 xl:mt-5"
+          >
+            Regístrate
+          </Link>
+
```

Comment:
Nit: this line can be erased

Reply:
Done: what is Nit in this case ? XD:

Reply:
Nit is short for "nitpicking", which meant someone is looking at really small details. In code reviews, the word refers to "I notice this small issue, its not that important but please check"